web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -947,7 +973,20 @@ private Object validateStudyEvent(StudyEventDataBean studyEventDataBean, StudySu
     }
 
 
-    public ErrorObj validateStartAndEndDateAndOrder(StudyEventDataBean studyEventDataBean) {
+    private boolean isEventCrfCompleted(EventCrf eventCrf) {
+    	if(eventCrf.getStatusId() == EventCRFStatus.INITIAL_DATA_ENTRY_COMPLETE.getCode()) {
+    		return true;
+    	}else if(eventCrf.getStatusId() == EventCRFStatus.DOUBLE_DATA_ENTRY_COMPLETE.getCode()) {
+    		return true;
+    	}else if(eventCrf.getStatusId() == 2) {
```

Comment:
Do not use numbers for statuses, use enum names instead, it will be clearer to others to read the code and understand .

Reply:
did code changes based on code review